### PR TITLE
update: added bsc zksync ethereum chains

### PR DESCRIPTION
**Pull request by tharunhack**

### CODE REVIEW

File: aggregators/kanalabs/index.ts [hunk 1]
```diff
@@ -23,9 +23,9 @@ const contract: IContract = {
   [CHAIN.ARBITRUM]: "0xA1BB807fF6701f4e1b404Eb35Da55d8E9f3fb25c",
   [CHAIN.AVAX]: "0xA1BB807fF6701f4e1b404Eb35Da55d8E9f3fb25c",
   [CHAIN.POLYGON]: "0xA1BB807fF6701f4e1b404Eb35Da55d8E9f3fb25c",
-//   [CHAIN.BSC]: "0xA1BB807fF6701f4e1b404Eb35Da55d8E9f3fb25c",
-//   [CHAIN.ETHEREUM]: "0xA1BB807fF6701f4e1b404Eb35Da55d8E9f3fb25c",
-//   [CHAIN.ZKSYNC]: "0x18f79872b0255f7B57a131890739539B0Ad6ad4E",
+  [CHAIN.BSC]: "0xA1BB807fF6701f4e1b404Eb35Da55d8E9f3fb25c",
+  [CHAIN.ETHEREUM]: "0xA1BB807fF6701f4e1b404Eb35Da55d8E9f3fb25c",
+  [CHAIN.ZKSYNC]: "0x18f79872b0255f7B57a131890739539B0Ad6ad4E",
```
Comment (dtmkeng) [inline aggregators/kanalabs/index.ts]: is it zksync or zksync era ?

Comment (tharunhack) [reply aggregators/kanalabs/index.ts]: it is zksync era

Comment (tharunhack) [reply aggregators/kanalabs/index.ts]: @dtmkeng

Comment (dtmkeng) [reply aggregators/kanalabs/index.ts]: should set `era` instead `ZKSYNC`

Comment (tharunhack) [reply aggregators/kanalabs/index.ts]: I have changed to zksync to era

Comment (tharunhack) [reply aggregators/kanalabs/index.ts]: @dtmkeng

**Pull request merged**